mackup/main.py

```diff
@@ -15,6 +15,7 @@
 Options:
   -h --help     Show this screen.
   -f --force    Force every question asked to be answered with "Yes".
+  -r --root     Run as root.
```

Comment:
Can you make the doc more explicit? e.g. you PR title: "Allow Mackup to be run as a superuser".

Reply:
done as well.